huxley/api/tests/__init__.py

```diff
@@ -97,7 +97,7 @@ def assertInvalidEmailFormat(self, response, field):
 
     def assertInvalidPhone(self, response, field):
         self.assertEqual(response.data, {
-            u'non_field_errors': [u'This is an invalid phone number.']})
+            '%s' % field: [u'This is an invalid phone number.']})
```

Comment:
This is the formatting used elsewhere in the file so it was used for consistency.